app/src/main/assets/applozic-settings.json

```diff
@@ -181,5 +181,6 @@
   "chatBarTopLineViewColor": "",
   "isMultipleAttachmentSelectionEnabled": false,
   "isImageCompressionEnabled": false,
-  "minimumCompressionThresholdForImagesInMB": 5
+  "minimumCompressionThresholdForImagesInMB": 5,
+  "hideAttachmentOptionsBeforeHandover": false
```

Comment:
change the flag to `hideattachmentoptionswithbots`

Reply:
Okay